docker/docker-compose.yml

```diff
@@ -58,10 +58,7 @@ x-sbtc-signer: &sbtc-signer
     - sbtc-signer
 
 x-stacks-signer: &stacks-signer
-  build:
-    context: stacks
-    args:
-      GIT_COMMIT: *STACKS_BLOCKCHAIN_COMMIT
+  image: blockstack/stacks-core:3.1.0.0.5
```

Comment:
nit: we could repurpose `STACKS_BLOCKCHAIN_COMMIT` to ensure we use the same version for the node and the signers.

Reply:
Ahhh, kind of as a global tag. I think doing something like this supported, I'll try it out.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1307/commits/d83e29c25b2d880e6e7af07e49651981d772156e

Reply:
I couldn't repurpose that variable since I don't think docker compose supports that kind of thing. It also might be a yaml limitation. I went with a common stanza instead, where we only need to update a single place when updating the blockchain.